### PR TITLE
Note about sampled transactions in Discover

### PR DESCRIPTION
**Pull request by ale-cota**
## Description of changes

Added note to the Search section to explain that searches based on transaction data may be subject to sampling.

Screenshot 2023-11-03 at 11 44 52

## Pre-merge checklist

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

**Review by matejminar — APPROVED** [review]

File: src/docs/product/discover-queries/query-builder.mdx [hunk 1]
```diff
@@ -58,6 +58,10 @@ Click on any of these sections to further refine your search. For example, click
 
 ![Facet map for transaction and handled tags. Map looks like a bar with gradient colors.](./discover-facet-map.png)
 
+<Note>
+When performing searches on transaction events, based on your search criteria and sample rate, the events available may be limited because Discover uses sampled data only. Learn more by reading about [retention priorities](/product/performance/retention-priorities/).
+<\Note>
```
Comment (ale-cota) [inline src/docs/product/discover-queries/query-builder.mdx:63]:
```suggestion
</Note>
```

File: src/docs/product/discover-queries/query-builder.mdx [hunk 2]
```diff
@@ -58,6 +58,13 @@ Click on any of these sections to further refine your search. For example, click
 
 ![Facet map for transaction and handled tags. Map looks like a bar with gradient colors.](./discover-facet-map.png)
 
+<Note>
+  When performing searches on transaction events, based on your search criteria
+  and sample rate, the events available may be limited because Discover uses
+  sampled data only. Learn more by reading about [retention
```
Comment (vivianyentran) [inline src/docs/product/discover-queries/query-builder.mdx:64]:
```suggestion
Discover only uses sampled data for transactions. This means that when performing searches on transaction events, due to your search criteria and sample rate, you might see limited results. Learn more by reading about ...
```
Is it true that Discover only uses sampled data? Are there cases where it uses all data?

**Review by vivianyentran — APPROVED** [review]
Thanks for adding the note! Added a suggestion

File: src/docs/product/discover-queries/query-builder.mdx (hunk 2 above)
Comment (ale-cota) [reply src/docs/product/discover-queries/query-builder.mdx:64]: For transactions it's always sampled, but for low volumes we happen to sample 100% 😅 That's clarified further in the docs about retention priorities, so we can also rephrase it to how you suggested 👍
